django/urls/resolvers.py

```diff
@@ -331,6 +334,34 @@ def check(self):
             )
         return warnings
 
+    def _check_pattern_unmatched_angle_brackets(self):
+        warnings = []
+        segments = self._route.split("/")
```

Comment:
Can we avoid splitting on `/`? From the perspective of this check there's nothing special about this character in the URL.

Reply:
Unmatched angle brackets cannot cross `/` as it raises `ImproperlyConfigured`: _"... uses parameter name ... which isn't a valid Python identifier."_

Reply:
Yes, that's why I suggest we don't split on it, as the `RoutePattern` can't exist in a state where such invalid characters exist between `<` and `>`?

Reply:
> Damn @felixxm merged.

I regret nothing :wink:

Reply:
![IMG_4777](https://github.com/django/django/assets/857609/45258c52-34dc-4da8-9c64-60e35b439381)